README.md

```diff
@@ -26,11 +25,11 @@ int main()
 
     TabSample tab;
     // explicit all_of(tab)
-    for(const auto& row : db.run(select(all_of(tab)).from(tab).where(true)))
+    for(const auto& row : db(select(all_of(tab)).from(tab).where(true)))
     {
         int64_t alpha = row.alpha;
-        bool beta = row.beta;
-        std::string gamma = row.gamma;
+        std::string beta = row.beta;
+        bool gamma = row.gamma;
```

Comment:
To match the definitions in [TabSample.h](https://github.com/rbock/sqlpp11-connector-sqlite3/blob/master/tests/TabSample.h#L107)

Guess I should update the table definition at line 20/21 as well?